dsviz/render.go

```diff
@@ -113,3 +162,51 @@ func readEntries(reader dsio.EntryReader) (interface{}, error) {
 	}
 	return array, nil
 }
+
+const (
+	bite = 1 << (10 * iota)
+	kilobyte
+	megabyte
+	gigabyte
+	terabyte
+	petabyte
+	exabyte
+	zettabyte
+	yottabyte
+)
+
+func printByteInfo(n int64) string {
```

Comment:
This was copied from cmd/print.go, right? The key difference there is that the input argument has type `int`, not `int64`. Doing it this way makes line 182 (`l := int64(n)`) a no-op. The argument should be `int` so that, on 32-bit platforms like arm6 Raspberry Pi, the `int` will be `int32` and line 182 (`l := int64(n)`) will upcast it to a 64-bit integer.